app/views/tag/contributors.html.erb

```diff
@@ -68,7 +68,7 @@
 
   <% end %>
 
-  <% if @notes.nil? || @notes.length == 0 %>
+  <% if @users.nil? || @users.length == 0 %>
```

Comment:
Found some unexpected behaviour when @notes.length was 0 but @users still existed hence modified the condition a bit!!